src/modules/peek/Peek.Common/Extensions/DisposablePropertyStore.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using Peek.Common.Models;
+
+namespace Peek.Common.Extensions
+{
+    public sealed class DisposablePropertyStore : IDisposable
+    {
+        private IPropertyStore? _propertyStore;
```

Comment:
I dont think it need to be nullable since you init it in the constructor.

Reply:
Updated. Also VS suggested me to put it as readonly so I did